hack/cherry_pick_pull.sh

```diff
@@ -33,11 +33,25 @@ REGENERATE_DOCS=${REGENERATE_DOCS:-""}
 UPSTREAM_REMOTE=${UPSTREAM_REMOTE:-upstream}
 FORK_REMOTE=${FORK_REMOTE:-origin}
 
+
+set +o errexit
+GIT_LOCAL_USER=$(git config --local --get user.name)
```

Comment:
"user.name" in your git config is not guaranteed to match your github username

Reply:
yes. Looks like we can not get the github username directly. If user set "user.name" consistent with github username, can we use this to avoid of set the environment.